Tests/MongoSwiftSyncTests/LoadBalancerTests.swift

```diff
@@ -0,0 +1,65 @@
+import MongoSwiftSync
+import Nimble
+import TestsCommon
+
+final class LoadBalancerTests: MongoSwiftTestCase {
+    let skipFiles: [String] = [
+        // We don't support this option.
+        "wait-queue-timeouts.json"
+    ]
+
+    func testLoadBalancers() throws {
+        let tests = try retrieveSpecTestFiles(
+            specName: "load-balancers",
+            excludeFiles: skipFiles,
+            asType: UnifiedTestFile.self
+        ).filter { $0.0 == "cursors.json" }.map { $0.1 }
+
+        let skipTests = [
```

Comment:
unfortunately a lot of caveats here, but we still do get to run a decent number of the tests, so it seems better than nothing. let me know if all the reasoning below makes sense.